llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -4142,6 +4142,32 @@ static bool canNarrowLoad(VPWidenRecipe *WideMember0, unsigned OpIdx,
   return false;
 }
 
+static bool canNarrowOps(ArrayRef<VPValue *> Ops) {
+  SmallVector<VPValue *> Ops0;
+  auto *WideMember0 = dyn_cast<VPWidenRecipe>(Ops[0]);
```

Comment:
Looks like this doesn't quite match the old code in `narrowInterleaveGroups`, i.e. 

```
  ... = dyn_cast_or_null<VPWidenRecipe>(
        InterleaveR->getStoredValues()[0]->getDefiningRecipe());
```

Does it matter? Just a bit worried it's not actually NFC and changing behaviour in a subtle way.